go/vt/vtgate/logstats.go

```diff
@@ -125,6 +127,15 @@ func (stats *LogStats) Logf(w io.Writer, params url.Values) error {
 		return nil
 	}
 
+	// FormatBindVariables call might panic so we're going to catch it here and print out the stack trace to help us
```

Comment:
Doesn't an uncaught panic already print a stack trace? Or is the problem that someone else has been recovering the panic somewhere else in the call stack?

Reply:
You're right that an uncaught panic already prints a stack trace.  @sougou had asked me offline to use `recover()` to handle the place where the panic was occurring as an extra safety net. I was hoping he would comment here on the PR to clarify the full extent of what he wanted to achieve with the `recover()`.  I've yet to have an opportunity to sync up with him about this part.

Reply:
I heard that @sougou can be found at the PlanetScale booth at KubeCon. You should try to find him there. :)

Reply:
Chatted with him.  He meant without an explicit panic and deepthi clarified that vtgate by design should never panic.  Correcting now

Reply:
Adjusted https://github.com/vitessio/vitess/pull/5441/commits/c4270d29e53a88a6c0366909f0efa0513c4c387f.  PTAL